src/components/common/ProjectSelect/ProjectSelect.tsx

```diff
@@ -0,0 +1,13 @@
+import type { FC } from 'react';
+import { ModalHOC } from 'src/modules/Providers/modals/ModalHOC/ModalHOC';
+
+import ProjectSelectTypeAhead from '@components/common/ProjectSelect/ProjectSelectTypeAhead.tsx';
+import type { ProjectSelectProps } from '@components/common/ProjectSelect/types.ts';
+
+const ProjectSelect: FC<ProjectSelectProps> = ({ ...props }) => (
+  <ModalHOC>
```

Comment:
I'm a little confused looking back at this change now why a ModalHOC was used. Maybe you could explain this to me.

Reply:
Oh got it. I forgot there's a modal associated with the create project ability now, my bad. Thanks for clarifying!